docs/tutorials/tools.md

```
@@ -1,33 +1,41 @@
+and perform specific operations. Think of tools as callable functions that bridge the gap between
+natural language understanding and system execution.
 
-RAI offers a comprehensive set of pre-built tools, including both general-purpose and ROS 2-specific tools [here](../../src/rai_core/rai/tools/ros2). However, in some cases, you may need to develop custom tools tailored to specific robots or applications. This guide demonstrates how to create custom tools in RAI using the [LangChain framework](https://python.langchain.com/docs/).
+RAI offers a comprehensive set of pre-built tools, including both general-purpose and ROS 2-specific
+tools [here](../../src/rai_core/rai/tools/ros2). However, in some cases, you may need to develop
+custom tools tailored to specific robots or applications. This guide demonstrates how to create
+custom tools in RAI using the [LangChain framework](https://python.langchain.com/docs/).
 
 RAI supports two primary approaches for implementing tools, each with distinct advantages:
 
 ### `BaseTool` Class
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)